realm/realm-library/src/main/java/io/realm/log/AndroidLogger.java

```diff
@@ -1,150 +0,0 @@
-/*
- * Copyright 2016 Realm Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package io.realm.log;
-
-import android.util.Log;
-
-import static android.util.Log.getStackTraceString;
-
-/**
- * Logger implementation outputting to Android LogCat.
- * Androids {@link Log}levels are mapped to Realm {@link LogLevel}s using the following table:
- *
- * <table summary="Comparison of Realm and Android log levels">
- * <tr>
- *     <td>{@link LogLevel#ALL}</td><td>{@link Log#VERBOSE}</td>
- *     <td>{@link LogLevel#TRACE}</td><td>{@link Log#VERBOSE}</td>
- *     <td>{@link LogLevel#DEBUG}</td><td>{@link Log#DEBUG}</td>
- *     <td>{@link LogLevel#INFO}</td><td>{@link Log#INFO}</td>
- *     <td>{@link LogLevel#WARN}</td><td>{@link Log#WARN}</td>
- *     <td>{@link LogLevel#ERROR}</td><td>{@link Log#ERROR}</td>
- *     <td>{@link LogLevel#FATAL}</td><td>{@link Log#ERROR}</td>
- *     <td>{@link LogLevel#OFF}</td><td>Not supported. Remove the logger instead.</td>
- * </tr>
- * </table>
- */
-public class AndroidLogger implements Logger {
```

Comment:
This class is in the public API, I don't think we can remove it in a minor release.